core/commands/id.go

```diff
@@ -217,7 +216,6 @@ func printSelf(keyEnc ke.KeyEncoder, node *core.IpfsNode) (interface{}, error) {
 		info.Protocols = node.PeerHost.Mux().Protocols()
 		sort.Slice(info.Protocols, func(i, j int) bool { return info.Protocols[i] < info.Protocols[j] })
 	}
-	info.ProtocolVersion = identify.DefaultProtocolVersion
```

Comment:
If you unset this you should remove from the `IdOutput` stuct definition.

Also what is this exactly ? I mean it's always `ipfs/0.1.0` but how is that used what happen if I change it to `ipfs/0.2.0` by accident ?

Reply:
It's not, it's a "protocol version" field in the Protobuf that used to be checked for a match a very long time ago, but is not any more.

Reply:
It can still be sent and might be useful for debug purposes (maybe?), so it should be kept on the `IdOutput`.

Reply:
If it has no sementics anymore let's remove it and accept the breaking change of `ipfs id` (I would be surprised if anyone depends on this field, and if they do sounds easy to stop doing so given it hasn't done anything intresting in a while).

Reply:
Done.